core/src/main/java/hudson/model/Queue.java

```diff
@@ -723,7 +752,7 @@ private synchronized WaitingItem peek() {
         return itemsView.get();
     }
 
-    public synchronized Item getItem(int id) {
```

Comment:
Balls ... I did test this to make sure the `@AdaptField` still worked but this is a different issue that was not checked for (and was not expected).